src/e2e/controllers.e2e.js

```diff
@@ -11,6 +11,26 @@ beforeAll(async () => {
   });
   page = await browser.newPage();
   await page.goto('http://localhost:8000/dashboard/#/');
+  // Login using dummy credentials
```

Comment:
We need to move these e2e tests to the `describe()` section as individual tests below as the `beforeAll()` section only handles initialization of the test environment.

Reply:
@gurbirkalsi there was issue with the submit button id in each test file and in `search.e2e.js`, it wasn't getting redirected to search page. I have fixed both of them.